plugin/trino-hive/src/test/java/io/trino/plugin/hive/s3/TestTrinoS3FileSystemAccessOperations.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hive.s3;
+
+import com.google.common.collect.HashMultiset;
+import com.google.common.collect.ImmutableMultiset;
+import com.google.common.collect.Multiset;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.testing.exporter.InMemorySpanExporter;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
+import io.trino.plugin.hive.containers.HiveMinioDataLake;
+import io.trino.testing.AbstractTestQueryFramework;
+import io.trino.testing.DistributedQueryRunner;
+import io.trino.testing.QueryRunner;
+import org.intellij.lang.annotations.Language;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import static io.trino.plugin.hive.util.MultisetAssertions.assertMultisetsEqual;
+import static java.util.stream.Collectors.toCollection;
+
+public class TestTrinoS3FileSystemAccessOperations
+        extends AbstractTestQueryFramework
+{
+    private InMemorySpanExporter spanExporter;
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        HiveMinioDataLake hiveMinioDataLake = closeAfterClass(new HiveMinioDataLake("test-bucket"));
+        hiveMinioDataLake.start();
+
+        spanExporter = closeAfterClass(InMemorySpanExporter.create());
+
+        SdkTracerProvider tracerProvider = SdkTracerProvider.builder()
+                .addSpanProcessor(SimpleSpanProcessor.create(spanExporter))
+                .build();
+
+        OpenTelemetry openTelemetry = OpenTelemetrySdk.builder()
+                .setTracerProvider(tracerProvider)
+                .build();
+
+        return S3HiveQueryRunner.builder(hiveMinioDataLake).setOpenTelemetry(openTelemetry).build();
+    }
+
+    @AfterClass(alwaysRun = true)
+    public void tearDown()
+    {
+        spanExporter = null; // closed by closeAfterClass
+    }
+
+    @Test
+    public void testSelectWithFilter()
+    {
+        assertUpdate("CREATE TABLE test_select_from_where AS SELECT 2 AS age", 1);
```

Comment:
It looks like we run with default Hive format (currently ORC)
I think for this test we probably want to pin the format, and we definitely want to cover Parquet.
Let's provide format as a `@DataProvider`
Yes, it will result in a bit ugly test code, but ensuring test coverage stays consistent for both, and that we see differences (if any) clearly.